Gemfile.lock

```diff
@@ -786,7 +786,7 @@ DEPENDENCIES
   byebug
   capybara
   coffee-rails (~> 4.1.0)
-  config
+  config (~> 1.2)
```

Comment:
The version of config that was installed hasn't been changed. It's good that we're being more specific with what we want though.